examples/fodo_channel_restart/input_fodo.in

```diff
@@ -0,0 +1,60 @@
+###############################################################################
+# Particle Beam(s)
+###############################################################################
+beam.npart = 10000
+beam.units = static
+beam.kin_energy = 2.0e3
+beam.charge = 1.0e-9
+beam.particle = electron
+beam.distribution = waterbag_from_twiss
+beam.alphaX = -1.5905003499999992
+beam.alphaY = -beam.alphaX
+beam.alphaT = 0.0
+beam.betaX = 2.8216194100262637
+beam.betaY = beam.betaX
+beam.betaT = 0.5
+beam.emittX = 2e-09
+beam.emittY = beam.emittX
+beam.emittT = 2e-06
+
+
+###############################################################################
+# Beamline: lattice elements and segments
+###############################################################################
+lattice.elements = monitor drift1 quad1 drift2 quad2 drift3
+lattice.nslice = 5
+lattice.periods = 101  # FODO channel of 101 periods
```

Comment:
Unlike `run_fodo.py`, I don't see user input for a restart in the app input file.  Is the intention to add the capability of restart via the app input also?